MANUAL_REGRESSION.md

```diff
@@ -709,6 +709,17 @@ Given user opened the link with `?uploadFallback=false` flag
    - You should see the document selector screen
 8. You should be able to complete the ID verification flow successfully
 
+### 37. Interrupt flow if selfie fallback is deactivated and MediaRecorder is not supported
+
```

Comment:
Should specify what device OS versions/browsers in general would not have MediaRecorder support & this UX is expected on as Applause testers don't necessarily know this. See earlier Face flow tests in this docs for reference.

Reply:
updated! :)